src/dash-table/components/Table/Table.less

```diff
@@ -141,337 +142,353 @@
 }
 
 .dash-table-container {
-
     .previous-next-container {
-	display: inline-block;
-	float: right;
-	padding: 5px 0px;
-
-
-	.page-number {
-	    font-family: monospace;
-	    display: inline-block;
-	    .last-page {
-		min-width: 30px;
-		display: inline-block;
-		text-align: center;
-		margin: 0.5em;
-	    }
-	}
-
-	input.current-page {
-	    display: inline-block;
-	    border-bottom: solid lightgrey 1px !important;
-	    color: black;
-	    border: none;
-	    width: 30px;
-	    text-align: center;
-	    font-family: monospace;
-	    font-size: 10pt;
-	    margin: 0.5em;
-
-	    &::placeholder {
-		color: black;
-	    }
-	    
-	    &:focus {
-		outline: none;
-		
-		&::placeholder {
-		    opacity: 0;
-		}
-	    }
-	}
-
-	button.previous-page, button.next-page, button.first-page, button.last-page {
-	    transition-duration: 400ms;
-	    padding: 5px;
-	    border: none;
-	    display: inline-block;
-	    margin-left: 5px;
-	    margin-right: 5px;
-
-	    &:hover {
-		color: hotpink;
-		
-		&:disabled {
-		    color: graytext
-		}
-	    }
+        display: inline-block;
```

Comment:
Indentation here was all weirded out - fixed. Lots of noise.

Reply:
Looks like some got fixed and some other parts got mucked up. Mostly lines 507+ but there are a few before that that look weird as well. Is there an auto-formatter for `.less`?

Reply:
OIC you have some hard tabs mixed in with spaces here - this file looks funny in #728 as well as a result.

Reply:
I'll double check to make sure this file has been cleaned up.. but yes, at some point it became a mix of tabs/spaces